tsconfig.base.json

```diff
@@ -55,6 +55,7 @@
       "@fluentui/react-progress": ["packages/react-components/react-progress/src/index.ts"],
       "@fluentui/react-provider": ["packages/react-components/react-provider/src/index.ts"],
       "@fluentui/react-radio": ["packages/react-components/react-radio/src/index.ts"],
+      "@fluentui/react-recipes": ["packages/react-components/react-recipes/src/index.ts"],
```

Comment:
Not sure what makes a package a candidate for app, but the only concern I had about making it an app is that it won't actually have an app. It would only be a placeholder that stores the recipes. If that's not an issue, I would say we should!

Reply:
Actually, now that I think about it, it would be nice to have some sort of preview outside of the docsite. Especially while they are not going to be shown in the docs.